src/renderer/prefComponents/imageUploader/index.vue

```diff
@@ -108,11 +126,30 @@ export default {
       })
     },
     setCurrentUploader (value) {
+      const service = services[value]
+      if (!service) {
+        console.error(`Cannot find service "${value}"!`)
+        return
+      }
+
+      const legalNotices = document.getElementsByClassName(`pref-cb-legal-notices ${value}`)
```

Comment:
How about use component props to pass some state to child component, if you really want to get the DOM, use `this.$refs`. https://vuejs.org/v2/api/#ref